src/components/constants/Constants.js

```diff
@@ -0,0 +1,14 @@
+export default {
+    MAIN_PAGE: 'mainPage',
+    DETAILS_PAGE: "detailPage",
+
+    DEFAULT_LIMIT: 12,
+    RELEASE_DATE: 'release date',
+    RATING: 'rating',
+
+    TITLE: 'title',
+    GENRES: 'genres',
+
+    MOVIES: 'movies',
+    DESC: 'desc'
```

Comment:
It's better to export each member separately to don't prevent tree shaking

Reply:
done